test/webflow_test.rb

```diff
@@ -94,6 +94,23 @@ def test_handles_raises_validation_errors
     end
   end
 
+  def test_it_raises_validation_errors_with_problems
+    VCR.use_cassette('test_raises_validation_errors_with_problems') do
+      data = {
+        _archived:            false,
+        _draft:               false,
+        name:                 'SomeName',
+        field_with_validation: "sh\nrt",
+      }
+      begin
+        client.create_item(COLLECTION_ID, data)
```

Comment:
If this line doesn't raise, this (and preview test) will pass, without actually testing anything.